nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/RunNiFi.java

```diff
@@ -1056,7 +1056,7 @@ private void waitForShutdown(final String pid, final Logger logger, final File s
             logger.error("Failed to delete pid file {}; this file should be cleaned up manually", pidFile);
         }
 
-        logger.info("NiFi has finished shutting down.");
+        logger.info("NiFi has finished shutting down. (PID={})", pid);
```

Comment:
References to the Process ID appear differently in this class versus `ShutdownHook`.  Recommend keeping `PID` capitalized and adjusting the log as follows:
```suggestion
        logger.info("NiFi process [{}] shutdown completed", pid);
```